js/ui/interaction.js

```diff
@@ -36,6 +36,7 @@ function Interaction(el) {
     scrollwheel(zoom);
     el.addEventListener('dblclick', ondoubleclick, false);
     window.addEventListener('resize', resize, false);
+    window.addEventListener('keydown', keydown, false);
```

Comment:
Hmm, I'm not sure to be honest, blame says it was like this in the original contribution.